src/kit/Elevation/index.module.scss

```diff
@@ -0,0 +1,169 @@
+/* stylelint-disable scss/percent-placeholder-pattern */
+
+:root {
+  /* elevation bg */
+  --elevation-0-bg-dark: #1c1c1c;
+  --elevation-1-bg-dark: #242424;
+  --elevation-2-bg-dark: #2a2a2a;
+  --elevation-3-bg-dark: #323232;
+  --elevation-4-bg-dark: #383838;
+  --elevation-0-bg-light: #f0f0f0;
+  --elevation-1-bg-light: #f7f7f7;
+  --elevation-2-bg-light: #fdfdfd;
+  --elevation-3-bg-light: #fff;
+  --elevation-4-bg-light: #fff;
+
+  /* elevation hover */
+  --elevation-0-hover-dark: #1c1c1c;
+  --elevation-1-hover-dark: #323232;
+  --elevation-2-hover-dark: #303030;
+  --elevation-3-hover-dark: #303030;
+  --elevation-4-hover-dark: #303030;
+  --elevation-0-hover-light: #ececec;
+  --elevation-1-hover-light: #f5f5f5;
+  --elevation-2-hover-light: #f8f8f8;
+  --elevation-3-hover-light: #f8f8f8;
+  --elevation-4-hover-light: #f8f8f8;
+
+  /* elevation border */
+  --elevation-0-border-dark: #323232;
+  --elevation-1-border-dark: #323232;
+  --elevation-2-border-dark: #3a3a3a;
+  --elevation-3-border-dark: #3a3a3a;
+  --elevation-4-border-dark: #3a3a3a;
+  --elevation-0-border-light: #e2e2e2;
+  --elevation-1-border-light: #e2e2e2;
+  --elevation-2-border-light: #e8e8e8;
+  --elevation-3-border-light: #e8e8e8;
+  --elevation-4-border-light: #e8e8e8;
+
+  /* CSS classes for elevation 0 */
+  %elevation-0-light {
+    background-color: var(--elevation-0-bg-light);
+
+    &%with-border {
+      border: 2px solid var(--elevation-0-border-light);
```

Comment:
fix: unless there's a specific reason to do otherwise these all should probably use `--theme-stroke-width` for border width